### ISSUE TITLE
alert level name shouldn't be hardcoded

### ISSUE THREAD

**Issue by lzk228** [space-wizards/space-station-14#21525]
title

**Comment by dvir001** [comment]
On what level is the issue?
UI in computers?

Got any place you seen it hardcoded to have someone to migrate it to FTL?

**Comment by lzk228** [comment]
@dvir001 iirc, game uses ID of alert level to find a locale string
for example:
```
newlevel:
      announcement: alert-level-violet-announcement
      sound: /Audio/Misc/notice1.ogg
      color: Violet
      emergencyLightColor: Violet
      shuttleTime: 600

game searching `alert-level-newlevel` in locale
that issue is little similar to #18425

**Comment by dvir001** [comment]
Is it a big issue if you can still rename them on FTL anyway?
alert-levels.ftl

**Comment by lzk228** [comment]
if you can't change that basic things with only yml changes, that's an issue

**Comment by dvir001** [comment]
> if you can't change that basic things with only yml changes, that's an issue

Alright, so just to be on the same page, on what level do you want it changed?

**Comment by lzk228** [comment]
> > if you can't change that basic things with only yml changes, that's an issue
> 
> Alright, so just to be on the same page, on what level do you want it changed?

that's not matter, i just want name datafield in alertelev prototype

**Comment by deltanedas** [comment]
you can still translate it so theres no real need for it

**Comment by lzk228** [comment]
hardcode is bad

**Comment by deltanedas** [comment]
its not hardcoded

**Comment by IProduceWidgets** [comment]
I agree it would be better to have it read a datafield in the prototype and then just have the datafield point to the ftl.

This is useful for upload prototypes. However, I also think it should be done for the sake of consistency with all the other prototypes in the game which use string datafields (that many times but not always point to a locale ftl)
